tests/frontend/dahlia/invoke-memory.fuse

```diff
@@ -0,0 +1,9 @@
+def mem_copy(dest: ubit<32>[1], src: ubit<32>[1]) = {
```

Comment:
Can we make this test use memory of size 5 or something and copy in a loop? I am slightly suspicious of access address `0` since every signal defaults to `0`.

Reply:
Yeah I can do that. Not sure what you mean by the suspicions? Given that this does an add of two `2 x 4` memories, I'm pretty confident it is correct.

FWIW, I hand-wrote most of the futil version.

Reply:
Ah, I’m suspicious of the FuTIL compiler too :).

Reply:
Ok I've updated both the Dahlia frontend and the futil correctness tests to use memories of size 5.